dali/operators/sequence/optical_flow/optical_flow.cc

```diff
@@ -19,38 +19,56 @@ namespace dali {
 
 
 DALI_SCHEMA(OpticalFlow)
-                .DocStr(R"code(Calculates the Optical Flow for sequence of images given as a input.
-Mandatory input for the operator is a sequence of frames.
-As an optional input, operator accepts external hints for OF calculation.
-The output format of this operator matches the output format of OF driver API.
-Dali uses Turing optical flow hardware implementation: https://developer.nvidia.com/opticalflow-sdk
+                .DocStr(R"code(Calculates the optical flow for a sequence of images that were
+provided as the input.
+
+The mandatory input for the operator is a sequence of frames. As an Optionally, the operator
+also accepts external hints for the optical flow calculation. The output format of this operator
+matches the output format of the optical flow driver API.
+Refer to https://developer.nvidia.com/opticalflow-sdk for more information about the
+Turing and Ampere optical flow hardware that is used by DALI. This operator allows sequence inputs.
```

Comment:
```suggestion
The main input for the operator is a sequence of frames. Optionally, the operator
can be provided with external hints for the optical flow calculation. The output format of this operator
matches the output format of the optical flow driver API.
Refer to https://developer.nvidia.com/opticalflow-sdk for more information about the
Turing and Ampere optical flow hardware that is used by DALI.
```
The info about sequences is rather redundant.

Reply:
Done